GliaWidgets/View/Chat/Message/Content/ChoiceCard/ChoiceCardOptionStyle.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
```

Comment:
MUIC-287 Add borders to choice cards options
This refactors choice cards' styles a bit to make them more customizable and more consistent with other styles